.github/workflows/update.yml

```diff
@@ -52,3 +52,66 @@ jobs:
               exit 1
             fi
           fi
+
+  nix-flake-submodules:
+    name: 'Nix flake sumbodule sync'
+    runs-on: ubuntu-latest
+    steps:
+      - id: config
+        run: |
+          ref=${{ github.ref }}
+          if [ "${{ github.event_name }}" == 'pull_request' ]; then
+            ref="${{ github.event.pull_request.head.sha }}"
+          fi
+          ref="${ref#refs/heads/}"
+          echo "::set-output name=ref::$ref"
+          git config --global user.name github-actions
+          git config --global user.email github-actions@github.com
```

Comment:
Might be better if the commits come from a runtimeverification email. I'd recommend devops@runtimeverification.com